data/json/items/armor/suits_clothes.json

```diff
@@ -918,6 +918,52 @@
     "material_thickness": 1,
     "flags": [ "VARSIZE", "HOOD" ]
   },
+  {
+    "id": "wool_suit_bunny",
+    "type": "ARMOR",
+    "name": { "str": "bunny onesie" },
+    "description": "An encumbering and fluffy bunny onesie with a hood and adorned with a pair of bunny ears.  Very fluffy.",
+    "weight": "883 g",
+    "volume": "3 L",
+    "price": "80 USD",
+    "price_postapoc": "3 USD 50 cent",
+    "material": [ "wool" ],
+    "symbol": "[",
+    "looks_like": "touring_suit",
+    "color": "light_gray",
+    "armor": [
+      { "covers": [ "torso", "leg_l", "leg_r" ], "coverage": 100, "encumbrance": 20 },
+      { "covers": [ "arm_l", "arm_r" ], "coverage": 90, "encumbrance": 20 },
+      { "covers": [ "head" ], "coverage": 80, "encumbrance": 15 },
+      { "covers": [ "foot_l", "foot_r" ], "coverage": 100, "encumbrance": 15 }
+    ],
+    "warmth": 50,
+    "material_thickness": 1,
+    "flags": [ "VARSIZE", "HOOD" ]
+  },
+  {
+    "id": "wool_suit_bunny",
+    "type": "ARMOR",
+    "name": { "str": "bunny onesie" },
+    "description": "An encumbering and fluffy bunny onesie with a hood and adorned with a pair of bunny ears.  Very fluffy.",
+    "weight": "883 g",
+    "volume": "3 L",
+    "price": "80 USD",
+    "price_postapoc": "3 USD 50 cent",
+    "material": [ "wool" ],
+    "symbol": "[",
+    "looks_like": "touring_suit",
+    "color": "light_gray",
+    "armor": [
+      { "covers": [ "torso", "leg_l", "leg_r" ], "coverage": 100, "encumbrance": 20 },
+      { "covers": [ "arm_l", "arm_r" ], "coverage": 90, "encumbrance": 20 },
+      { "covers": [ "head" ], "coverage": 80, "encumbrance": 15 },
+      { "covers": [ "foot_l", "foot_r" ], "coverage": 100, "encumbrance": 15 }
+    ],
+    "warmth": 50,
+    "material_thickness": 1,
+    "flags": [ "VARSIZE", "HOOD" ]
+  },
```

Comment:
This is the same as the one above it?

Reply:
I mean they are completely identical, even their `id`s are identical.

Reply:
Fine, i'll go make it a variant, then. It's gonna be a bit annoying in my part since i also have to rename the sprites for these onesies

Reply:
Oh, i just realized that theres two bunny onesie jsons in there. Still imma go turn it into a variant

Reply:
Done.